Makefile

```diff
@@ -97,6 +97,15 @@ test-gomemcache:
 	  docker stop gomemcache-integ ; \
 	fi
 
+.PHONY: test-gorm
+test-gorm:
```

Comment:
why `test-gorm` is implemented as "integration tests" ?
it does not require any Docker container as a dependency. could it not be a simple unit test like most of the other instrumentation?

Reply:
i dont remember, i think it was failing to run on the ci due to sqlite dependencies

Reply:
The tests are not running on CI anyway because `INTEGRATION` env var is not set: https://github.com/open-telemetry/opentelemetry-go-contrib/pull/508/checks?check_run_id=3055645335

Reply:
After fixing https://github.com/open-telemetry/opentelemetry-go-contrib/pull/508#discussion_r668631032 and `util.IntegrationShouldRun("test-gorm")` everything worked fine on my machine so I hope that it does not need to be an "integration test" 😉

Reply:
i removed this, lets see if tests will pass